etc/google-earth.profile

```diff
@@ -0,0 +1,39 @@
+# Firejail profile for google-earth
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/google-earth.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ${HOME}/.config/Google
+noblacklist ${HOME}/.googleearth 
```

Comment:
See above - may need to rework the noblacklist?

Reply:
Fixed, I think. Thanks.